.cirrus.yml

```diff
@@ -620,8 +620,18 @@ task:
 
   only_if: $CIRRUS_CHANGE_MESSAGE !=~ '.*\nci-os-only:.*' || $CIRRUS_CHANGE_MESSAGE =~ '.*\nci-os-only:[^\n]*(macos|darwin|osx).*'
 
-  osx_instance:
-    image: monterey-base
+  matrix:
+    - name: macOS - Monterey - Meson - ARM CPU
```

Comment:
This currently sets the number of CPUs - but they don't have that many. So it seems we should just remove that?

Reply:
Instead of removing it, setting it to 4 makes more sense to me since it is used in 'BUILD_JOBS' and 'TEST_JOBS'.